packages/obonode/obojobo-chunks-multiple-choice-assessment/mc-assessment-explanation.js

```diff
@@ -30,17 +32,26 @@ export default class MCAssessmentExplanation extends React.Component {
 		const isShowingExplanation = this.props.isShowingExplanation
 		const solutionModel = this.props.solutionModel
 		const SolutionComponent = solutionModel.getComponentClass()
-
+		const readExplanationLabel = solutionModel.attributes.content.readExplanationLabel
```

Comment:
Instead of grabbing this from attributes.content you should take it from `solutionModel.modelState`, which means you'll need to set it on modelState in the obonode/obojobo-chunks-multiple-choice-assessment/adapter.js.

I also think we need to add another label, which I didn't think about until playing with it. When viewing the full explanation there's a gray label that shows which says "Full Explanation:". That's set via CSS with a :before pseudo element. That probably has to change to an actual element, and then add an option to also set that label as well.

Reply:
Update on above - instead of adding another label and dealing with the psuedo element I think we could just remove the psuedo element to handle all cases.